utils/db/healthCheck.ts

```diff
@@ -0,0 +1,31 @@
+import conn from ".";
+
+export const dpuHealthCheckStatus = async (topicId: string): Promise<boolean> => {
+	console.debug(`[dpuHealthCheckStatus] Getting health check status for topic ${topicId}`);
+	const query = `SELECT dpu.online
```

Comment:
Why are we creating a new table for this? Can we not just keep this information in the `users` table? We only have one `topic_name` for each `user.id`. 

As long as there is even one active docker listening to that `topic_name`, we just have to return `true`. 

I just want to understand what is it that we are trying to optimize for by creating tables for every new thing?

Reply:
Maybe in the future, we might want to add a feature which can inform the user if they have multiple DPUs running redundantly. Even then, I don't see any real utility in creating a separate table. It will only make my queries to the database more complex.

Reply:
I am not trying to create new table for every new thing. I wanted to create a new table for dpu where I can track whether it is online, where it might be in the setup process, any other dpu related information we might want to track. We can definitely stuff it all in users table, if it is so difficult to create queries with a new table. I wanted to logically separate data so we have lesser tech debt going forward. I'll just create a new column in users for dpu health status

Reply:
Let's discuss this over a call. (Please note that whatever I am saying are just suggestions. You can override them if you think they are not correct. You have spent more time on this and would have thought more deeply about it - I am just trying to help.)